src/index.html

```diff
@@ -15,8 +18,97 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#">
+        <img
+          class="img"
+          src="images/logo.png"
+          alt="MOYO_main_logo"
```

Comment:
The `href` attribute in the anchor tag should not be empty or contain just '#'. If this is a placeholder, consider adding a comment or using a more descriptive placeholder. Also, the `alt` attribute should describe the image content, 'MOYO_main_logo' could be more descriptive to improve accessibility and SEO.